### DISCUSSION TITLE
Turn off printing

### DISCUSSION THREAD

**Discussion by ignaspakamore** [PasaOpasen/geneticalgorithm2-6.8.7#17] [Show and tell]
Hi, 

I was wondering is there a way to deactivate printing? I am ruining this code on several instances simultaneously and its a mess, so was thinking if it would be possible to deactivate it?

**Comment by PasaOpasen** [comment]
@ignaspakamore hello!

do u mean disabling all prints?

**Reply by ignaspakamore** [reply]
Yes, it would be great if the code would run silently in the background without printing anything.

**Reply by PasaOpasen** [reply]
@ignaspakamore okay, I will solve it in 1-2 days

**Reply by ignaspakamore** [reply]
Amazing, thank you very much!!!

**Comment by PasaOpasen** [comment]
@ignaspakamore now there are `disable_progress_bar` and `disable_printing` arguments

see an [example](https://github.com/PasaOpasen/geneticalgorithm2/blob/master/tests/disable_messages_and_printing.py)

**Reply by ignaspakamore** [reply]
Thanks a lot!
